web/src/beta/utils/file.ts

```diff
@@ -40,3 +41,30 @@ export async function fetchAndZipFiles(
 
   return file;
 }
+
+const CHUNK_SIZE = 16 * 1024 * 1024; // 16MB
+
+export const uploadFile = async (file: File, teamId: string, axios: any) => {
```

Comment:
Could you rename the function to be more specific? I think it's not a general file upload, it has a certain end point as `/split-import` so it's for split import project only.

Reply:
Or better rename & move this to projectApi.ts, api is abstracted, it could use gql or rest.

Reply:
Do we still need this function after you move it to projectApi?
If you want to make a general upload file with chunks function here please expose the formData property & endpoint & other configs as params of this function.